### ISSUE TITLE
what happened to the GUI-variants of LinkHut*?

### ISSUE THREAD

**Issue by umlaeute** [Ableton/link#91]
release v3.0.3 has dropped the GUI-variants of `LinkHut` and ´LinkHutSilent`.
what for?

the [release notes](https://github.com/Ableton/link/releases/tag/Link-3.0.3) only mention:

> 
> - Fixed an issue that could lead to memory corruption during deallocation
> - Fixed a bug that would lead to incorrect times for SessionState::timeForIsPlaying()
> - Allow injecting a custom clock into Link
> 

i can't seem to see any mention of removing any utilities.

assuming `link` follows [semantic versioning](https://semver.org/), the last release would have been a *bugfix* release (for "backwards compatible bugfixes"), which would also not indicate that any binaries are removed.

### why am I complaining?
mostly because the package for Debian (and Ubuntu and other derivatives) used to include those GUI-variants, and now they are gone - and I'm wearing my Debian maintainer hat.

also I think that the examples are not only mere reference uses, but are actually useful.

**Comment by fgo-ableton** [comment]
Sorry. I forgot to mention that in the release notes -I updated them. I actually wasn't aware the examples are part of the package.
We removed the Qt dependency because maintaining it on CI was a bit cumbersome. The CLI application has the same functionality now, as the GUI apps had.
Regarding semantic versioning: The API of Link itself should still be compatible, and I consider the examples being part of documentation.
